testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -62,13 +62,25 @@ Feature: Smoke tests for <client>
     And I enter the hostname of "<client>" as "target"
     And I click on preview
     Then I should see a "Target systems (1)" text
-    When I wait until I do not see "pending" text
-    And I click on run
+    And I wait until I do not see "pending" text
+    When I click on run
     And I wait until I see "show response" text
     And I expand the results for "<client>"
     Then I should see "My remote command output" in the command output for "<client>"
 
-  #TODO: Run a remote command on traditional clients
+@skip_for_minion
```

Comment:
Here we need to install the package that allows this (not installed by default mgr-cfg-actions) and run the command via ssh on the client first `mgr-actions-control --enable-all` which allows the UI to have the tab there. Then we can run this scenario.

Reply:
This is already done in the initialization of each traditional client.

Reply:
You are absolutely correct, sorry